dashboard/src/routes/(alternatives)/t/$treeName/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { createFileRoute, redirect } from '@tanstack/react-router';
+
+export const Route = createFileRoute('/(alternatives)/t/$treeName/')({
+  loaderDeps: ({ search }) => ({ search }),
+  loader: ({ deps, params }) => {
+    throw redirect({
+      to: '/tree/$treeName',
```

Comment:
```suggestion
      to: '/tree/$treeName/',
```

that way tou don't need to create that  extra `route`

when it finishes with `/` is a leaf

Reply:
When I put this code, I receive a type error and it suggests `/tree/$treeName`

Reply:
The reason behind this is that `/tree/$treeName` does not have an index, only a route. To solve this you can create the index for `/tree/$treeName` or keep `to: '/tree/$treeName'. Maybe the former would be the correct approach here, I'm not so sure it's needed though